extensions/ql-vscode/package.json

```diff
@@ -1380,6 +1380,7 @@
     "@types/sinon-chai": "~3.2.3",
     "@types/stream-chain": "~2.0.1",
     "@types/stream-json": "~1.7.1",
+    "@types/tar-stream": "^2.2.2",
```

Comment:
It looks like we're just bringing in the types module, do we need the actual module too? 🤔

Reply:
Good point! Although it's already included as a transitive dependency, I'll also add it as a dependency itself too.